DnDCharacterSheet/Models/Coin.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Models
+{
+    public class Coin
+    {
+        [Key]
```

Comment:
i'd avoid having your domain class related to any external representation (think EF entity) by removing this relation you can enjoy a cleaner domain without any references to it's external dependencies (think infrastructure)